source/tests/test_player.d

```diff
@@ -2,21 +2,16 @@ module test_player;
 
 public import player;
 
-import dunit;
-
-class TestPlayer
+unittest
 {
-  mixin UnitTest;
+  import dunit.toolkit;
+
+  auto player = Player.getMock("jim", "green");
+  player.disableParentMethods();
```

Comment:
What's the point of making this mockable and disabling parent methods? We don't extend any classes so no parent methods should exist.